Update README.md with data description

noelgoodwin opened the pull request:
Update README.md with data description

noelgoodwin committed:
Update README.md with data description

noelgoodwin committed:
Fix typos in description

jonmgeiger committed:
Fix italics and reduce redundancy

jonmgeiger commented:
Great work @noelgoodwin! It might be good to include, if you can find information on this, how the condition estimates were recorded and/or why they're estimates rather than exact numbers. I seem to remember talking in class about how they just surveyed all households within a school district rather than taking into account whether there were actually students who went to school in that household.

noelgoodwin committed:
Clarify estimate descriptions

I added a blurb about how conditions estimates were made. Most of them actually do account for whether there are school aged children in the household, and I made note of the ones don't in their descriptions.
